tests/components/training/losses_test.py

```diff
@@ -275,3 +288,99 @@ def test_mapg_loss(
 
     assert loss_critic == 4.5
     assert low_loss_critic < loss_critic
+
+
+def test_mapg_huber_loss(
+    mock_trainer: Trainer,
+    mapg_loss: MAPGWithTrustRegionClippingLoss,  # noqa: E501
+) -> None:
+    """Test whether mapg huber loss output is as expected"""
+    huber_loss = HuberValueLossFunction()
```

Comment:
Can we set `huber_delta` in the test? Incase the default ever changes (we still want this test to pass).

Reply:
Yeah sure!

Reply:
I made a test to check that it update the value in the config: https://github.com/instadeepai/Mava/blob/451b5f896b07610eee46c7487dd743d5c81f25b0/tests/components/training/losses_test.py#L175

Reply:
Thanks!